src/preact.d.ts

```diff
@@ -277,8 +284,8 @@ declare namespace JSX {
 		charSet?:string;
 		challenge?:string;
 		checked?:boolean;
-		class?:string;
-		className?:string;
+		class?:string | Object;
+		className?:string | { [key:string]: string | boolean | number };
```

Comment:
oh yeah. dang. That's my bad. That should be:

```javascript
className?: string | number | { [key:string]: boolean };
```

And I guess `class?` should be the same